plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -159,12 +164,26 @@ func (*CloudWatch) SampleConfig() string {
 }
 
 func (c *CloudWatch) Connect() error {
+
 	cfg, err := c.CredentialConfig.Credentials()
+
+	if err != nil {
+		return err
+	}
+
+	ctx := context.Background()
+	client, err := c.HTTPClientConfig.CreateClient(ctx, c.Log)
+
 	if err != nil {
 		return err
 	}
 
-	c.svc = cloudwatch.NewFromConfig(cfg)
+	c.svc = cloudwatch.NewFromConfig(cfg, func(options *cloudwatch.Options) {
+		// Disable logging
+		options.ClientLogMode = 0
```

Comment:
It looks like disabling logging is unrelated to this PR's goal of adding proxy http connections. Is this necessary or could we just accept NewFromConfig's default?